### PR TITLE
add --insecure option to check-jenkins.rb and metrics-jenkins.rb

### PR DESCRIPTION
**Pull request by sabbene**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [ ] Update Changelog following the conventions laid out on [Our CHANGELOG Guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

**Review by lexical-scope — APPROVED** [review]
+1

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ This CHANGELOG follows the format listed at [Our CHANGELOG Guidelines](https://g
 Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- check-jenkins.rb: Add --insecure parameter to allow self signed ssl certs
```
Comment (majormoses) [inline CHANGELOG.md:8]: You should add a `### Added` header

**Review by majormoses — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (majormoses) [reply CHANGELOG.md:8]: I fixed it up for you prior to release.

**Comment by majormoses** [comment]
release: https://rubygems.org/gems/sensu-plugins-jenkins/versions/1.6.0

**Pull request merged**